### ISSUE TITLE
Summary tab tables not aligned vertically

### ISSUE THREAD

**Issue by oleiade** [grafana/xk6-dashboard#57]
This is me nitpicking, but in the summary tab, the bottom tables are arranged on two columns, within which the left and right tables have different heights. 

This leads to the bottom of the left table needing to be aligned with the bottom of the right one. Visually, I found it somewhat hard to parse and look a bit funky. I would suggest keeping a single column throughout the whole page instead.

![Screenshot 2023-08-28 at 16 49 46](https://github.com/grafana/xk6-dashboard/assets/690867/cdd9263b-5e12-4070-8eb0-13b122e879b9)

What do you think?

**Comment by szkiba** [comment]
Agree. (this is what I do on @print media, so probably on screen should do the same)

**Comment by szkiba** [comment]
Fixed in [v0.6.0](https://github.com/grafana/xk6-dashboard/releases/tag/v0.6.0)
